src/style.css

```diff
@@ -0,0 +1,38 @@
+body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+  justify-content: center;
+
+}
+
+.stars-item {
+  background-image: url("images/star.svg");
+  background-repeat: no-repeat;
+  background-position: center;
+  margin-right: 4px;
+  width: 16px;
+  height: 16px;
+}
+
+.stars--1 > .stars-item:nth-child(-n + 1) {
+  background-image: url("images/star-active.svg");
+}
+
+.stars--2 > .stars-item:nth-child(-n + 2) {
+  background-image: url("images/star-active.svg");
+}
+
+.stars--3 > .stars-item:nth-child(-n + 3) {
+  background-image: url("images/star-active.svg");
+}
+
+.stars--4 > .stars-item:nth-child(-n + 4) {
+  background-image: url("images/star-active.svg");
+}
+
+.stars--5 > .stars-item:nth-child(-n + 5) {
+  background-image: url("images/star-active.svg");
+}
```

Comment:
rewrite this code using string `background-image: url("images/star-active.svg");` only once